public/editor-client/src/defaultTemplates/utils.ts

```diff
@@ -1,387 +1,81 @@
-import { flatten, uniq, upperFirst } from "lodash";
+import { upperFirst } from "lodash";
 import {
-  APIPopup,
-  BlockWithThumbs,
   Categories,
-  Kit,
-  KitType,
-  Style
+  LayoutsAPI,
+  LayoutTemplateWithThumbs,
+  StoriesAPI,
+  StoriesTemplateWithThumbs
 } from "../types/DefaultTemplate";
-import { pipe } from "../utils/fp/pipe";
 
-type CatTypes = Kit | APIPopup;
+const BLANK_PAGE = "Blank";
 
-export const getUniqueKitCategories = (collections: CatTypes[]): Categories[] =>
-  pipe(
-    (collections: CatTypes[]) =>
-      collections.map((collection: CatTypes) => collection.categories),
-    flatten,
-    uniq,
-    (cats) =>
-      cats.map((cat) => ({
-        title: upperFirst(cat),
-        slug: cat,
-        id: cat
-      }))
-  )(collections);
-
-export const getUniqueKitTypes = (collections: Kit[]): KitType[] =>
-  pipe(
-    (collections: Kit[]) => collections.map((collection) => collection.theme),
-    flatten,
-    uniq,
-    (uni) =>
-      uni.map((u) => ({
-        title: upperFirst(u),
-        id: u,
-        name: u,
-        icon: "nc-light"
-      }))
-  )(collections);
-
-export const converterKit = (
-  kit: Kit[],
-  url: string,
-  kitId: string
-): {
-  blocks: BlockWithThumbs[];
-  categories: Categories[];
-  types: KitType[];
-} => {
-  const categories = getUniqueKitCategories(kit);
-  const types = getUniqueKitTypes(kit);
-
-  const blocks: BlockWithThumbs[] = kit.map(
+export const convertLayouts = (
+  collections: LayoutsAPI[],
+  thumbUrl: string
+): LayoutTemplateWithThumbs[] =>
+  collections.map(
     ({
-      slug,
+      title,
       categories,
+      color,
+      pagesCount,
       pro,
-      thumbnail,
       keywords,
       thumbnailWidth,
-      thumbnailHeight
-    }) => ({
-      id: slug,
-      cat: [categories],
-      title: slug,
-      type: types[0].name,
-      keywords: keywords ?? "",
       thumbnailHeight,
-      thumbnailWidth,
-      thumbnailSrc: `${url}${thumbnail}`,
-      pro: pro === "yes",
-      kitId
-    })
+      thumbnail,
+      slug
+    }) => {
+      return {
+        name: title,
+        cat: categories.split(",").map((item) => item.trim()),
+        color,
+        pagesCount: Number(pagesCount),
```

Comment:
better use `Num.read` instead of `Number(pagesCount)`, because `Number()` can return `NaN`

![image](https://github.com/ThemeFuse/Brizy/assets/52622998/d48c87d6-ce6f-460a-8e66-cac114871b43)